python/spec/fixtures/psycopg_homepage.html

```diff
@@ -1,229 +1,319 @@
+        <i class="fa-li fa-lg fa fa-book" aria-hidden="true"></i>
+        Documentation</a>
+    </li>
+    <li>
+      <a href="https://github.com/psycopg/psycopg2/">
+        <i class="fa-li fa-lg fa fa-github" aria-hidden="true"></i>
+        Source code</a>
+    </li>
+    <li>
+      <a href="https://github.com/psycopg/psycopg2/issues">
+        <i class="fa-li fa-lg fa fa-bug" aria-hidden="true"></i>
+        Bug tracker</a>
+    </li>
+    <li>
+      <a href="https://pypi.org/project/psycopg2/">
+        <i class="fa-li fa-lg fa fa-cloud-download" aria-hidden="true"></i>
+        Download</a>
+    </li>
+  </ul>
 </div>
 
-<div class="clearbox">
-  <h3>Quick links</h3>
-  <ul>
-    <li><a href="http://initd.org/psycopg/tarballs/PSYCOPG-2-7/psycopg2-2.7.4.tar.gz"
-      >Download stable release (2.7.4)</a></li>
-
-    <li><a href="http://www.stickpeople.com/projects/python/win-psycopg/">Windows download</a></li>
-    <li><a href="https://github.com/psycopg/psycopg2/issues">Report a bug</a></li>
-    <li><a href="https://lists.postgresql.org/">Mailing list</a></li>
+<div class="dashed-top">
+  <h3>psycopg3</h3>
+
+  <ul class="fa-ul links">
+    <li>
+      <a href="/psycopg3/">
+        <i class="fa-li fa-lg fa fa-cogs" aria-hidden="true"></i>
+        The Project</a>
+    </li>
+    <li>
+      <a href="/psycopg3/docs/">
+        <i class="fa-li fa-lg fa fa-book" aria-hidden="true"></i>
+        Documentation</a>
+    </li>
+    <li>
+      <a href="https://github.com/psycopg/psycopg3/">
+        <i class="fa-li fa-lg fa fa-github" aria-hidden="true"></i>
+        Source code</a>
+    </li>
+    <li>
+      <a href="https://github.com/psycopg/psycopg3/issues">
+        <i class="fa-li fa-lg fa fa-bug" aria-hidden="true"></i>
+        Bug tracker</a>
+    </li>
   </ul>
 </div>
 
+<div class="dashed-top">
+  <h3>Mailing List</h3>
+
+  <ul class="fa-ul links">
+    <li>
+      <a href="https://lists.postgresql.org/">
+        <i class="fa-li fa-lg fa fa-pencil" aria-hidden="true"></i>
+        Subscribe</a>
+    </li>
+    <li>
+      <a href="https://www.postgresql.org/list/psycopg/">
+        <i class="fa-li fa-lg fa fa-envelope-o" aria-hidden="true"></i>
+        Archives</a>
+    </li>
+  </ul>
+</div>
 
-  </div>
+<div class="dashed-top">
+  <a class="twitter-timeline" data-width="300" data-height="600" data-dnt="true"
+    href="https://twitter.com/psycopg?ref_src=twsrc%5Etfw">Tweets by psycopg</a>
+  <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
 </div>
 
-<div class="clearBoth"></div>
+        </div>
 
-<div id="footer">
-  <p>&#169; 2010&#8212;2017 &#8212; Daniele Varrazzo (at gmail.com)
-</div>
+      </div>
+    </div>
+
+    <footer>
+      <div style="margin-bottom:0">
+        <p>© Copyright 2010—2021 by Daniele Varrazzo (at gmail), The Psycopg Team.</p>
+      </div>
+    </footer>
 
-</body>
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"></script>
```

Comment:
And of course CodeQL flags this new URL as untrusted source... 🤦 

I missed it when I was originally skimming the homepage HTML looking for such URLs... 

Oh well, might as well keep this PR even though it doesn't solve this issue it still modernizes the fixture. And at least here it's pulled in via HTTPS rather than HTTP...